### PR TITLE
Create Readme_zh_cn.md

### PR DESCRIPTION
**Pull request by 2271007263A**
### User  Documents for _Chinese Users_

### CODE REVIEW

**Comment by 2271007263A** [comment]
Added some content based on the original usage document.

- added content about bucket

I hope to add more payment methods, such as _WeChat Pay_ and _Alipay_.

（Use Google translation and Baidu translation）

**Comment by Befod** [comment]
split    单任务连接数

下载时使用 N 个连接. 如果提供超过 N 个 URI 地址, 则使用前 N 个地址, 剩余的地址将作为备用. 如果提供的 URI 地址不足 N 个, 这些地址多次使用以保证同时建立 N 个连接. 同一服务器的连接数会被 --max-connection-per-server 选项限制.

**Comment by niheaven** [comment]
Please note the white space between Chinese characters and English words. [pangu.js](https://github.com/vinta/pangu.js) may help.

**Comment by 2271007263A** [comment]
@niheaven Thanks

**Comment by 2271007263A** [comment]
已更新
谢谢

File: Readme_zh_cn.md [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+
+## Scoop 可以安装什么类型的应用程序?
+
+使用 Scoop 最适合安装的应用程序通常称为“便携式”应用程序：即在解压缩时独立运行的压缩程序文件，并且没有诸如更改注册表或将文件放在程序目录之外的副作用。
+
+由于安装程序很常见，Scoop 也支持它们（以及它们的卸载程序）。
+
+Scoop 在处理单文件程序和 Powershell 脚本方面也很出色。 这些甚至不需要压缩。 请参阅[ runat ](https://github.com/lukesampson/scoop/blob/master/bucket/runat.json)包作为示例：它实际上只是一个 GitHub 仓库。
+
+Scoop 默认仓库的要求jiaow苛刻，许多应用无法安装，因此，请手动添加 bucket，您可以通过`scoop bucket known`查询 Scoop 还能直接识别哪些 bucket。并通过`scoop bucket add bucket `的格式添加 bucket。
```
Comment (h404bi) [inline Readme_zh_cn.md]: Is `jiaow` a typo or for what? Please also add spaces between inline code and chinese, don't forget to review the whole document.

File: Readme_zh_cn.md [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+我们可以通过这样的方式来将社区维护的 bucket 添加至本机的 Scoop bucket 列表：
+
+```
+scoop bucket add <仓库名> <仓库地址>
+```
+
+再举个例子，比如添加「 dorado 」仓：
+
+```
+scoop bucket add dorado https://github.com/h404bi/dorado
```
Comment (h404bi) [inline Readme_zh_cn.md]: Some sections are not in the original English README document. I assume that you repost some of the text from sspai, but perhaps my bucket should not appear in the README document.

File: Readme_zh_cn.md [hunk 3]
```diff
@@ -0,0 +1,145 @@
+<p align="center">
```
Comment (h404bi) [inline Readme_zh_cn.md]: You may change the filename to `README_zh-CN.md`.

**Comment by Ash258** [comment]
Are these other language readmes somehow managed by Github? For example if you open repository and have locales set to Chinese, it will be automatically opened.

If no, i would rather suggest to make one universal multi-language (community driven) documentation (https://github.com/pakeweb/scoop.sh could be extended for these purposes) to restrict documentation on one page. Github wikis are good, but everyone can change it without approving, ... So moving documentation under custom webpage and repository with changeable multi language support would make more sense and would be more accessible for new users.

**Comment by h404bi** [comment]
@r15ch13 should close this, it's really a mess now.

**Pull request closed without merging**